doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,453 @@
+
+IPC radio firmware
+------------------
+
+* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system in all devices that contain MRAM, such as the nRF54H Series devices.
+
+Matter bridge
+-------------
+
+* Added:
+
+  * Support for the ``UniqueID`` attribute in the Bridged Device Basic Information cluster.
+  * Version 2 of the bridged device data scheme containing ``UniqueID``.
+  * Kconfig options :ref:`CONFIG_BRIDGE_MIGRATE_PRE_2_7_0 <CONFIG_BRIDGE_MIGRATE_PRE_2_7_0>` and :ref:`CONFIG_BRIDGE_MIGRATE_VERSION_1 <CONFIG_BRIDGE_MIGRATE_VERSION_1>` to enable migration from older data schemes.
+
+nRF Desktop
+-----------
+
+* Updated:
+
+  * The :ref:`nrf_desktop_settings_loader` to make the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` the default settings backend for all board targets that use the MRAM technology.
+    As a result, all :ref:`zephyr:nrf54h20dk_nrf54h20` configurations were migrated from the NVS settings backend to the ZMS settings backend.
+  * :ref:`nrf_desktop_watchdog` by adding the :ref:`zephyr:nrf54h20dk_nrf54h20` release configuration.
+  * Updated the configuration files of the :ref:`nrf_desktop_click_detector` (:file:`click_detector_def.h`) to allow them to be used even when Bluetooth LE peer control using a dedicated button (:ref:`CONFIG_DESKTOP_BLE_PEER_CONTROL <config_desktop_app_options>`) is disabled.
+  * The DTS description for board targets with a different DTS overlay file for each build type to isolate the common configuration that is now defined in the :file:`app_common.dtsi` file.
+    The :ref:`zephyr:nrf54h20dk_nrf54h20` board configuration has been updated.
+
+  * The :ref:`nrf_desktop_ble_conn_params` with the following changes:
+
+    * Fixed the Bluetooth LE connection parameters update loop (NCSDK-30261) that replicated if an nRF Desktop dongle without Low Latency Packet Mode (LLPM) support was connected to an nRF Desktop peripheral with LLPM support.
+    * The module now waits until a triggered Bluetooth LE connection parameters update is completed before triggering subsequent updates for a given connection.
+    * Improved the log to also display the information if USB is suspended.
+      The information is needed to determine the requested connection parameters.
+    * The module now uses non-zero Bluetooth LE peripheral latency while USB is suspended.
+      This is done to prevent peripheral latency increase requests from :ref:`nrf_desktop_ble_latency` on peripheral's end.
+    * The module reverts the USB suspended Bluetooth LE connection parameter update when USB cable is disconnected.
+
+  * The :ref:`nrf_desktop_ble_scan` to always use a connection interval of 10 ms for peripherals without Low Latency Packet Mode (LLPM) support if a dongle supports LLPM and more than one Bluetooth LE connection.
```

Comment:
nRF54H20 DK does not use `nrf_desktop_ble_conn_params` module. No support for dongle (Bluetooth Central role)

Reply:
will remove this